.github/workflows/test_cli.yml

```diff
@@ -0,0 +1,49 @@
+name: Validate neo-node Compilation with Local neo
+
+on:
+  push:
+    branches: [master]
```

Comment:
BTW, how about running these actions on each pull request? It may be useful to check the build immediately when PR is open.

Reply:
as @shargon said, this will lock the pr, since a pr can be merged only if all checks pass

Reply:
Maybe a neo PR requires a specific branch of neo-devpack to work

Reply:
> this will lock the pr, since a pr can be merged only if all checks have passed

I've just checked, we can specify a set of github actions in the repo's `Settings` -> `Branches` -> `Branch protection rule` -> `Require status checks to pass before merging `. There still might be some problems with running a specific version of core with Modules/DevPack/CLI/etc, that's why I suggest to consider enabling this action on every pull request and exclude this action from the list of "required to merge" checks. If we do this, then this action will be run each time PR is created, but it's not required to pass this check for PR merging. Thus, this action will be kind of informative "helper" for which it's allowed to fail.

@shargon, @Liaojinghui, what do you think?

Reply:
Here's the example of such setup:

![image](https://github.com/neo-project/neo/assets/22775709/84ffde40-815c-4b83-b8ca-ccf587d4c1dc)